tools/ansible/roles/dockerfile/templates/Dockerfile.j2

```diff
@@ -43,7 +43,7 @@ RUN dnf -y update && dnf install -y 'dnf-command(config-manager)' && \
     xmlsec1-devel \
     xmlsec1-openssl-devel
 
-RUN pip3 install virtualenv
+RUN pip3 install virtualenv setuptools_scm build
```

Comment:
The rest of the pip calls are using `python3 -m pip` . Should we align and do the same here?